elfpy/agents/policies/random_agent.py

```diff
@@ -0,0 +1,152 @@
+"""User strategy that opens or closes a random position with a random allowed amount"""
+
+import numpy as np
+from numpy.random._generator import Generator as numpyGenerator
+
+import elfpy
+import elfpy.markets.hyperdrive.hyperdrive_actions as hyperdrive_actions
+import elfpy.markets.hyperdrive.hyperdrive_market as hyperdrive_market
+import elfpy.agents.agent as agent
+import elfpy.types as types
+
+# pylint: disable=too-many-arguments
+# pylint: disable=duplicate-code
+
+
+class Policy(agent.Agent):
+    """Random agent"""
+
+    def __init__(self, rng: numpyGenerator, trade_chance: float, wallet_address: int, budget: int = 10_000) -> None:
+        """Adds custom attributes"""
+        self.trade_chance = trade_chance
+        self.rng = rng
+        super().__init__(wallet_address, budget)
+
+    def get_available_actions(
+        self,
+        disallowed_actions: "list[hyperdrive_actions.MarketActionType] | None" = None,
```

Comment:
prefer non quoted types.

Reply:
We get a lint error if we use non quoted types, and I think quoted types are better than lint ignore statements

Reply:
nvm -- just needed to import annotations. Fixed.